### PR TITLE
Eberly/search-patient

### PR DESCRIPTION
**Pull request by ieberl1**
**Feel free to discard any changes you don't like (not all are functional)**
- Added patient search
- Updated sidenav
- Updated pages (employee-creation,  patient-creation, view employee, view patient) to only be as tall as the content within (removed forcing height to "100vh")
- "Forgot Username/Password" is now underlined & cursor is a pointer
- Login is attempted when you press "ENTER"
-  Error message is displayed when UN or Pass fields are empty
-  "OPTIONS" -> "Options"
-  Employee & Patient search results now show cursor as a pointer when hovering over the rows
- Fixed bug w/ pagination where page > 0 but there were only a couple results (so nothing was displayed until you go back to page 0).

### CODE REVIEW

**Review by ieberl1 — COMMENTED** [review]
Looks good to me

File: src/app/search-employee/search-employee.component.ts [hunk 1]
```diff
@@ -62,14 +63,17 @@ export class SearchEmployeeComponent implements OnInit {
...
-  submitSearch() {
+  submitSearch(resetPage: boolean=false) {
     // Verify inputs are valid
     if( this.employeeid.hasError('invaliduuidv4') ||
         this.lastname.invalid ||
         this.dateofbirth.hasError('invalidDate') ){
       return
     }
-    console.log("here")
+    if (resetPage === true){
```
Comment (JosueMolinaMorales) [inline src/app/search-employee/search-employee.component.ts]:
```suggestion
    if (resetPage){
```

File: src/app/search-patient/search-patient.component.ts [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  }
+
+  submitSearch(resetPage: boolean=false) {
+    // Verify inputs are valid
+    if( this.patientid.hasError('invaliduuidv4') ||
+        this.lastname.invalid ||
+        this.dateofbirth.hasError('invalidDate') ){
+      return
+    }
+    if (resetPage === true){
```
Comment (JosueMolinaMorales) [inline src/app/search-patient/search-patient.component.ts]:
```suggestion
    if (resetPage){
```

**Review by JosueMolinaMorales — APPROVED** [review]
Great work! Small things. Going to pull this branch and check out the changes visually

**Pull request merged**